### PR TITLE
Add NU5127 docs

### PR DESCRIPTION
**Pull request by zivkan**
Pack warning 4/5 for NuGet/Home#8439

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[dacf4b6](https://github.com/NuGet/docs.microsoft.com-nuget/commits/dacf4b626249653f4499e482cc66e7c475a44050)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-nu5127/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1692) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-nu5127/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1692) |
[docs/reference/errors-and-warnings/NU5127.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-nu5127/docs/reference/errors-and-warnings/NU5127.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5127?branch=pr-en-us-1692) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C20%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909202131554963-1692%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nsJRFXhzgO62fJmu%2FgYeGP7RH54RjqGG6fVfx93uos8%3D&st=2019-09-20T21%3A28%3A00Z&se=2019-10-21T21%3A33%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5127.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+-lib/net45/_._
+-lib/netstandard2.0/_._</pre>
+
+### Issue
+
+Projects using packages with `PackageReference` only use `lib/` and `ref/` assemblies to determine package compatibility. Therefore, a package without any `lib/` or `ref/` files will be considered compatible with all projects. However, if that package contains build files specific to one or more [Target Framework Monikers (TFMs)](../target-frameworks.md), a package consumer may expect the package to fail if none of the build files are used.
+
+### Solution
+
+As the warning message suggests, create an empty file named `_._` in the lib folder for the TFMs listed. This will allow NuGet to fail the restore for `PackageReference` projects when the project is incompatible with the package.
```
Comment (kartheekp-ms) [inline docs/reference/errors-and-warnings/NU5127.md]: I think you are referring to creating a file with name `_._` with `no file extension` as part of the solution.

Comment (zivkan) [reply docs/reference/errors-and-warnings/NU5127.md]: Sorry, I don't understand. Is your comment that I should make it more obvious that the nupkg should contain this file?

The file's extension is `_`, so I really don't understand your no file extension comment. A package without `lib/` files is advanced and probably was not created by packing a project (although with our MSBuild pack targets it is possible, at least in SDK style projects), so there may not be a project or solution to add a `_._` file to. How the customer adds the `_._` depends on how they created the package and I hope they don't need instructions and worst case can copy how their `build/` files were added to the package.

**Comment by zivkan** [comment]
replaced by https://github.com/NuGet/docs.microsoft.com-nuget/pull/1697

**Pull request closed without merging**